voter-dapp/src/ActiveRequests.js

```diff
@@ -80,7 +120,9 @@ function ActiveRequests({ votingAccount, votingGateway }) {
     voteStatuses &&
     // Each of the subfetches has to complete. In drizzle, `undefined` means incomplete, while `null` means complete
     // but the fetched value was null, e.g., no `comittedValue` existed.
-    voteStatuses.every(voteStatus => voteStatus.committedValue !== undefined);
+    voteStatuses.every(voteStatus => voteStatus.committedValue !== undefined) &&
+    proposals &&
```

Comment:
Does this mean that we get a brief moment after the user hits "explain" where the page returns to "Looking up requests"?

This isn't a problem or anything, just curious. The only thing better than that might be to pop the dialogue immediately and show loading text there instead. However, I think detecting that exact situation below would be more effort than its worth.

Reply:
I don't think so actually.

The way I understand it, the page doesn't render until `subsequentFetchComplete` is `true`. The intention is that the the proposals are all fetched from the `Governor` before the page renders at all, but the decoding happens when you press `Explain`.

The hooks stuff is always confusing, so let me know if you think the code doesn't actually behave that like haha.

Reply:
Yup, reread, and I was conflating two separate functions in my head. I see what you're saying now. There's one loop that finds all the admin proposals in the list and requests them upfront. And the second function just maps the index to that prefetched data for display synchronously.

Makes sense.